tools/sync_test_server/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:16.04
+
+# Add realm repo
+RUN apt-get update -qq \
+    && apt-get install -y curl npm \
+    && curl -s https://packagecloud.io/install/repositories/realm/realm/script.deb.sh | bash \
+    && npm install winston temp httpdispatcher
+COPY keys/private.pem keys/public.pem configuration.yml /
+COPY ros-testing-server.js /usr/bin/
+# Install realm object server
+RUN apt-get update -qq \
+    && apt-get install -y realm-object-server-de=1.0.0-BETA-2.1-271
```

Comment:
Perhaps add a description in the README about which files that needs to be updated when we bump the ROS version? There seems to be at least 3 places right now